python/setup.py

```diff
@@ -37,7 +38,8 @@
                             '../cuML/external/ml-prims/external/cutlass',
                             '../cuML/external/cutlass',
                             '../cuML/external/ml-prims/external/cub',
-                            '/usr/local/cuda/include'],
+                            '/usr/local/cuda/include',
+                            numpy.get_include()],
```

Comment:
This was removed to avoid an abi conflict in cuDF, let’s not add it back until future testing shows we’re clear of that issue.

Reply:
The removal of the include was actually causing the python build to fail. Do you have any ideas why this might be happening?

Reply:
A remaining cimport numpy in one of your local cython files perhaps?

Reply:
@cjnolet your fork branch cloned freshly without the include numpy components in setup.py works perfectly, you probably have a rogue `cimport numpy` or other thing conflicting.